mobile/lib/modules/shared_link/views/shared_link_edit_page.dart

```diff
@@ -0,0 +1,337 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:immich_mobile/modules/shared_link/providers/shared_link.provider.dart';
+import 'package:openapi/api.dart';
```

Comment:
again, please try to avoid DTOs all across the app

Reply:
Updated.